### ISSUE TITLE
[HU2] Usuario con tiempo. Eliminar notas 

### ISSUE THREAD

**Issue by lentes4k** [lentes4k/Proyecto-IV#5]
[HU2] El usuario puede listar y decidir el borrado de aquellas notas que ya no use

**Comment by lentes4k** [comment]
Esto no lo entiendo. El usuario tras un tiempo usando la aplicación podrá borrar una nota. Cuándo se implemente la clase cajón con todos (o gran parte) de sus métodos tendrá acceso (mediante el método que se elija en un futuro ya que vamos a prescindir de la clase cajón en este objetivo) para borrar las notas que elija.
Supongo que el comentario va dirigido a la situación actual del proyecto. Pero tras mi última revisión del PR supongo que esta HU quedá suspendida hasta un futuro (cercano o no :smile: )

**Comment by JJ** [comment]
Insisto, ¿qué beneficio tiene borrando una nota? Si no obtiene ningún beneficio, no es una HU. Ya decidiréis a la hora de implementar qué hacer con las notas antiguas. El que la borre el usuario es una posibilidad, pero también que la borre el sistema o que haya un contexto en la cual el usuario la borre, por ejemplo, una lista de las notas que no se han consultado o algo así. En general, "borrar" es un detalle de implementación que no debería ir a una historia de usuario que, insisto, tienen siempre que incluir un beneficio para el cliente.

**Comment by lentes4k** [comment]
Obtiene el beneficio de tener más claridad, orden a fin de cuentas. Existe la posibilidad de que si el usuario no las elimina, cuando vaya a listar las notas se pierda entre información innecesaria para el mismo.
Si las notas antiguas las borra el sistema es posible que el usuario anotara información sobre el ordenador de casa de un familiar mayor(credenciales de acceso) que no usa hasta que dicho familiar le dice: "oye, ¿que contraseña le pusimos?" y al no usarla en mucho tiempo el sistema decidiera eliminarla

**Comment by JJ** [comment]
> Obtiene el beneficio de tener más claridad, orden a fin de cuentas. Existe la posibilidad de que si el usuario no las elimina, cuando vaya a listar las notas se pierda entre información innecesaria para el mismo. Si las notas antiguas las borra el sistema es posible que el usuario anotara información sobre el ordenador de casa de un familiar mayor(credenciales de acceso) que no usa hasta que dicho familiar le dice: "oye, ¿que contraseña le pusimos?" y al no usarla en mucho tiempo el sistema decidiera eliminarla

Pues eso es lo que tienes que tener en el user journey y la historia de usuario. Te insisto en todo caso que lo que hay que poner es el beneficio, "no tener información innecesaria". El cómo obtenerlo es lo que tienes que resolver. Puede ser borrando, puede ser creando cajones "antiguos", puede ser de miles de formas. El usuario sólo está interesado en el beneficio que obtiene, que siempre tiene que ser en el dominio del problema. Es decir, la HU dice lo que el usuario quiere. Aquí lo que tú estás diciendo es lo que tú quieres que haga el usuario

**Comment by JJ** [comment]
No. He dicho que escribáis el user journey. Y que no pongáis nada de implementación en las historias de usuario. El que el cajón se pueda borrar o no es un detalle de implementación. Lo que hay que esforzarse es en satisfacer al usuario, por lo que lo único que hay que decir es qué beneficios obtiene pero, sobre todo, el user journey que lleve hasta "querer tener más limpio y organizado todo".

**Comment by lentes4k** [comment]
Okey, procedo a eliminar esta HU

**Comment by JJ** [comment]
> Okey, procedo a eliminar esta HU

O puedes hacer caso a mis sugerencias, y simplemente postergarlo a un milestone posterior.

**Comment by lentes4k** [comment]
> > Okey, procedo a eliminar esta HU
> 
> 
> 
> O puedes hacer caso a mis sugerencias, y simplemente postergarlo a un milestone posterior.

Ah, haré eso. Es que pensaba que te referías a que no estaba aportando ningún beneficio y por la tanto no puede ser HU bajo ningún concepto
